datapackage_pipelines_mojp/common/processors/geocode.py

```diff
@@ -0,0 +1,123 @@
+from datapackage_pipelines.wrapper import ingest, spew
+from datapackage_pipelines_mojp.common.constants import GEOCODE_LOCATION_SCHEMA
+import requests, logging, datetime, os
+import geocoder
+from sqlalchemy import create_engine, MetaData, or_
+from sqlalchemy.orm import sessionmaker
+
+
+# attributes in entity details that might contain the full address in a single string
+LOCATION_ADDRESS_FIELDS = ["location"]
+
+# simple mapping of fields to their possible sources in the data
+# we will take the first item from the list of possible sources that has some data
+LOCATION_FIELDS_MAP = {"street": ["street"],
+                       "house_number": ["street_number", "house_number"],
+                       "city": ["city"],
+                       "country": ["country"],
+                       "po_box": ["pob", "zipcode", "postal_code", "pob_postal_code"],}
```

Comment:
I thought maybe to leave it incase we'd like to extend location to other items in the future, like a detailed address of a photo subject (e.g painting, synagogue, person's residence and so on). But I guess it could be added if and when needed.